addons/mail/static/src/thread/message.js

```diff
@@ -4,23 +4,45 @@ import { useMessaging } from "../messaging_hook";
 import { RelativeTime } from "./relative_time";
 import { Component } from "@odoo/owl";
 import { useService } from "@web/core/utils/hooks";
+import { MessageDeleteDialog } from '@mail/thread/message_delete_dialog';
 
 export class Message extends Component {
     static template = "mail.message";
     static components = { RelativeTime };
-    static props = ["message", "squashed?"];
+    static props = ["disableActions?", "message", "squashed?"];
```

Comment:
`hasActions` is a better name than `disableActions`. "disable" gives the impression that the actions are visible but in a disable state, which is not the case, as it's more their presence.